h2/settings.py

```diff
@@ -253,7 +253,18 @@ def __iter__(self):
     def __len__(self):
         return len(self._settings)
 
-
+    def __eq__(self, other):
+        if isinstance(other, Settings):
+            return self._settings == other._settings
+        else:
+            return False
+            
+    def __ne__(self, other):
+        if isinstance(other, Settings):
```

Comment:
I’d go one step further and drop the `isinstance()` call – we’re still duplicating that. Simply:

```python
def __ne__(self, other):
    return not self == other
```